iOS/MMIWG2S/ViewControllers/ViewController1.swift

```diff
@@ -0,0 +1,163 @@
+            imageView.widthAnchor.constraint(equalTo: view.widthAnchor, multiplier: 1),
+            imageView.heightAnchor.constraint(equalTo: view.heightAnchor),
+            imageView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            ])
+        
+        
+        let gradient = UIImage(named:"Rectangle1.png")
+        let gradientView = UIImageView(image:gradient)
+        gradientView.frame = view.frame
+        gradientView.contentMode = .scaleAspectFill
+        gradientView.clipsToBounds = true
+        view.addSubview(gradientView)
+        gradientView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            gradientView.topAnchor.constraint(equalTo: view.topAnchor),
+            gradientView.widthAnchor.constraint(equalTo: view.widthAnchor, multiplier: 1),
+            gradientView.heightAnchor.constraint(equalTo: view.heightAnchor),
+            gradientView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            ])
+
+        setupUI()
+    }
+    
+    func setupUI() {
+        
+        //Setup UI
+        let labelMMIW = UILabel(frame:CGRect(x:0,y:0,width:50,height:21))
+        let labelMMIWInfo = UILabel(frame:CGRect(x:0,y:0,width:200,height:200))
+        let buttonNext = UIButton(frame:CGRect(x:0,y:0, width:30, height:17))
+         
+        labelMMIW.translatesAutoresizingMaskIntoConstraints = false
+        labelMMIWInfo.translatesAutoresizingMaskIntoConstraints = false
+        buttonNext.translatesAutoresizingMaskIntoConstraints = false
+
+        view.addSubview(labelMMIW)
+        view.addSubview(labelMMIWInfo)
+
+        pc1View.translatesAutoresizingMaskIntoConstraints = false
+        pc2View.translatesAutoresizingMaskIntoConstraints = false
+        pc3View.translatesAutoresizingMaskIntoConstraints = false
+        pc4View.translatesAutoresizingMaskIntoConstraints = false
+    
+        labelMMIW.textAlignment = .center
+        labelMMIW.text = NSLocalizedString("slideviewheader", comment: "Text")
+        labelMMIW.textColor = UIColor.white
+        labelMMIW.font = UIFont(name: "Roboto-Regular", size:36.0 )
+
+        labelMMIWInfo.numberOfLines = 0
+        labelMMIWInfo.textColor = UIColor.white
+        labelMMIWInfo.font = UIFont(name: "Roboto-Regular", size:25.0 )
+        labelMMIWInfo.text = NSLocalizedString("slideview1text", comment: "text")
+        
+        pc1View.backgroundColor = UIColor.red
+        pc2View.backgroundColor = UIColor.white
+        pc3View.backgroundColor = UIColor.white
+        pc4View.backgroundColor = UIColor.white
+
+        buttonNext.alpha = 1.0
+        buttonNext.setTitleColor(UIColor.white, for:.normal)
+        buttonNext.setTitleColor(UIColor.green, for:.selected)
+
+        buttonNext.imageEdgeInsets = UIEdgeInsets(top:0, left:0, bottom:0, right:0 )
+        buttonNext.contentEdgeInsets = UIEdgeInsets(top:0, left:0, bottom:0, right:0 )
```

Comment:
I think what you want in both of these is `UIEdgeInsets.zero`. Are they not zero by default?